### PR TITLE
Add docker image for Ubuntu 18.04 for Helix with SqlServer installed

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

**Comment by smitpatel** [comment]
Related work item https://github.com/dotnet/core-eng/issues/9066

File: src/ubuntu/18.04/helixsqlserver/amd64/Dockerfile
Comment (MichaelSimons) [inline src/ubuntu/18.04/helixsqlserver/amd64/Dockerfile:50]: Consider consolidating these into a single instruction/layer.

File: src/ubuntu/18.04/helixsqlserver/amd64/Dockerfile
Comment (smitpatel) [reply src/ubuntu/18.04/helixsqlserver/amd64/Dockerfile]: Password will come from environment variable when starting the docker container.

**Comment by smitpatel** [comment]
Would there be a way for me to test out new images in in conjunction with https://github.com/dotnet/efcore azure pipelines before this PR is merged?

**Comment by MichaelSimons** [comment]
@smitpatel - You would have to build the images locally and push them to your own Docker registry.  I am not opposed to submitting the PR and iterating on the changes.

**Comment by smitpatel** [comment]
@MichaelSimons - I built image locally using build script from this repo and I was able to connect to SqlServer from EF Core tests inside the docker container. (Though I used the build script directly as opposed to what helix work item do)

File: src/ubuntu/18.04/helix/sqlserver/amd64/Dockerfile
Comment (mthalman) [inline src/ubuntu/18.04/helix/sqlserver/amd64/Dockerfile:18]: Is your intention really to use `CMD` or would `ENTRYPOINT` be more appropriate?  See https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact for the difference.

Comment (smitpatel) [reply src/ubuntu/18.04/helix/sqlserver/amd64/Dockerfile:18]: Either could work. This just starts SqlServer service.
I followed CMD from here https://github.com/microsoft/mssql-docker/blob/42c6ed8fa6d2a1696cf864e91e2660aeb7986dd0/linux/mssql-server-linux/Dockerfile#L15
Can change it to ENTRYPOINT too as this command should be run always for this particular image.

Comment (mthalman) [reply src/ubuntu/18.04/helix/sqlserver/amd64/Dockerfile:18]: It surprise me they do it that way.  But if that's how the official SQL Server image does it, might as well follow that pattern.

**Review by mthalman — APPROVED** [review]

**Comment by smitpatel** [comment]
@mthalman - Thanks for approval. I don't have write access in this repo so someone else has to merge this PR.

**Comment by mthalman** [comment]
I want to give @MichaelSimons a chance to sign off before merging.

**Review by MichaelSimons — APPROVED** [review]

**Pull request merged**
